yarn.lock

```diff
@@ -2779,6 +2779,11 @@
   resolved "https://registry.yarnpkg.com/@metamask/browser-passworder/-/browser-passworder-3.0.0.tgz#c06744e66a968ffa13f70cc71a7d3b15d86b0ee7"
   integrity sha512-hD10mgvhcDkZX8wnauw8udp1K2MzcbZfrN7Yon9sQ+OqVK9kiQ4VhZAyZNZTy9KJLtfoVD9Y2F6L4gEese7hDA==
 
+"@metamask/browser-passworder@^4.0.1":
```

Comment:
There is an entry in the policy override for `eth-keyring-controller>browser-passworder`. That will need to be updated to reference the new browser-passworder package name